BaseFilter/sections/specific.txt

```diff
@@ -579,6 +579,7 @@ $subdocument,third-party,denyallow=disqus.com,domain=welovemanga.one
 ! NOTE: Specific rules
 !
 !
+op.gg##tr.ad
```

Comment:
Maybe better apply to br.op.gg? I remember breakages on another subdomains by a good rules.